packages/swap-sdk/package.json

```diff
@@ -40,7 +40,7 @@
     "tiny-invariant": "^1.1.0",
     "tiny-warning": "^1.0.3",
     "toformat": "^2.0.0",
-    "viem": "^1.2.9"
+    "viem": "^1.10.9"
```

Comment:
let's add the `@pancakeswap/chains` as one of the dependencies to sdk. We can re-export the `ChainId` from our new pkg and mark it as deprecated